modules/@apostrophecms/area/ui/apos/components/AposAreaMenu.vue

```diff
@@ -164,6 +165,10 @@ export default {
       this.$emit('menu-open', e);
     },
     async add(name) {
+      // Potential TODO: If we find ourselves manually flipping these bits in other AposContextMenu overrides
+      // we should consider refactoring contextmenus to be able to self close when any click takes place within their el
+      // as it is often the logical experience (not always, see tag menus and filters)
+      this.$refs.contextMenu.isOpen = false;
```

Comment:
@abea I also changed this to be explicitly false (closed) because at this stage we always want it closed, truly toggling it felt overly cute and possibly bug prone